app/views/RoomsListView/hooks/useSubscriptions.test.tsx

```
@@ -0,0 +1,79 @@
+		render(
+			<Provider store={store}>
+				<Harness />
+			</Provider>
+		);
+
+		await waitFor(() => {
+			expect(queryCallCount).toBeGreaterThanOrEqual(1);
+		});
+		const initialCalls = queryCallCount;
+
+		await act(async () => {
```

ESLint

[error] Async arrow function has no 'await' expression.

(require-await)